tools/chibi-ffi

```diff
@@ -414,6 +415,23 @@
 (define (c-link lib)
   (set! *clibs* (cons lib *clibs*)))
 
+(define (c-flags-from-script cmd)
+  (eval '(import (chibi process)
+                 ; lest I clobber the existing string-split:
+                 (prefix (chibi string) imported:)
+                 (chibi char-set full)) ; for char-set:whitespace
```

Comment:
I think it's simpler to extend the existing string-split to support predicates and use char-whitespace?.